pyLDAvis/_prepare.py

```diff
@@ -89,13 +111,26 @@ def js_PCoA(distributions):
     Returns
     -------
     pcoa : array, shape (`n_dists`, 2)
-   """
-   dist_matrix = DistanceMatrix(dist.squareform(dist.pdist(distributions.values, _jensen_shannon)))
-   if skbio_old:
-       data = PCoA(dist_matrix).scores()
-       return data.site[:,0:2]
-   else:
-       return pcoa(dist_matrix).samples.values[:, 0:2]
+    """
+    dist_matrix = pairwise_distances(distributions, metric=_jensen_shannon)
```

Comment:
Is there anything that sklearn's `pairwise_distances` gets you that `scipy.spatial.distance.pdist` does not?

I ask because this one function is the one function being used from `sklearn` for the default `PCoA` option. If we move back to `pdist` here then we can make `sklearn` an optional dep just like we had it before.